HWLesson7/Task2.js

```diff
@@ -0,0 +1,18 @@
+const list = document.querySelector('#ingredients');
+
+const ingredients = [
+  'Картошка',
+  'Грибы',
+  'Чеснок',
+  'Помидоры',
+  'Зелень',
+  'Приправы',
+];
+
+const elements = ingredients.map(item => {
+  const li = document.createElement('li');
+  li.textContent = item;
+  return li;
```

Comment:
Можна одразу додавати елементи в даному циклі
```suggestion
  list.append(li);
```